lib/mc_forecast/simulation.rb

```diff
@@ -24,33 +24,50 @@ def run(init_state: nil, trials: 1_000, steps: 1, quantiles: [0.025, 0.16, 0.84,
 
     # Return an analysis of the events, containing:
     # { event_name:
-    #   { mean: [...], # per step
+    #   {
+    #     sum: { mean: ...,
+    #            quantiles:
+    #              { 0.025: ..., 0.975: ... }},
+    #     mean: [...], # per step
     #     quantiles:
     #      { 0.025: [...],
     #        0.975: [...]
     # }}}
     def analyze(events, quantiles)
```

Comment:
For plotting, we need quantiles sorted from high to low. Maybe include: `quantiles.sort` here?